### PR TITLE
Feature/feature iops 1903 add naming systems

### PR DESCRIPTION
**Pull request by SheenaSharma**

### CODE REVIEW

File: namingsystems/NamingSystem-England-ERSUBRN.xml
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-ERSUBRN.xml:1]: Add <?xml version="1.0" encoding="utf-8"?> to the top line for all files

File: namingsystems/NamingSystem-England-ERShealthcareserice.xml
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-ERShealthcareserice.xml:1]: ERShealthcareservice-> ERSHealthcareService

File: namingsystems/NamingSystem-England-ERShealthcareserice.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-ERShealthcareserice"/>
```
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-ERShealthcareserice.xml]: NamingSystem-England-ERShealthcareserice-> England-ERSHealthcareService [to be discussed]

File: namingsystems/NamingSystem-England-NhsMhsPartyKey.xml
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-NhsMhsPartyKey.xml:1]: NhsMhsPartyKey -> NHSMHSPartyKey

Comment (digitalhealthmartin) [reply namingsystems/NamingSystem-England-NhsMhsPartyKey.xml:1]: And apply to id/name appropriately

File: namingsystems/NamingSystem-England-NhsSpineASID.xml
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-NhsSpineASID.xml:1]: Nhs -> NHS

Comment (digitalhealthmartin) [reply namingsystems/NamingSystem-England-NhsSpineASID.xml:1]: Apply to id/name appropriately

File: namingsystems/NamingSystem-England-PrescriptionOrderNumberns.xml
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-PrescriptionOrderNumberns.xml:1]: Remove 'ns' at end of filename

Comment (digitalhealthmartin) [reply namingsystems/NamingSystem-England-PrescriptionOrderNumberns.xml:1]: And update id/name appropriately

File: namingsystems/NamingSystem-England-PrescriptionOrderNumberuuidns.xml
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-PrescriptionOrderNumberuuidns.xml:1]: Remove 'ns' at end and update uuid->UUID. Update id/name

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

File: namingsystems/NamingSystem-England-ERSUBRN.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-ERSUBRN" />
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-ERSUBRN.xml]: Why is the resource type-NamingSystem, included as part of the ID?
The convention is England-[BusinessNames]. Also, the business name should be more indicative of its meaning. Thus, this ID should read England-ERSUBRNumber.
This modification will also affect the file name.

File: namingsystems/NamingSystem-England-ERSUBRN.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <name value="NamingSystemEnglandERSUBRN" />
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-ERSUBRN.xml]: From above:  EnglandERSUBRNumber

File: namingsystems/NamingSystem-England-ERShealthcareserice.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <name value="NamingSystemEnglandERShealthcareserice"/>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-ERShealthcareserice.xml]: England-ERSHealthcareServiceID

File: namingsystems/NamingSystem-England-NhsMhsPartyKey.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-NhsMhsPartyKey" />
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-NhsMhsPartyKey.xml]: England-NHSMHSPartyKey

File: namingsystems/NamingSystem-England-NhsSpineASID.xml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-NhsSpineASID"/>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-NhsSpineASID.xml]: England-NHSSpineASID

File: namingsystems/NamingSystem-England-PrescriptionOrderItemNumber.xml [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-PrescriptionOrderItemNumber"/>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-PrescriptionOrderItemNumber.xml]: England-PrescriptionOrderItemNumber

File: namingsystems/NamingSystem-England-PrescriptionOrderNumberns.xml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-PrescriptionOrderNumberns"/>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-PrescriptionOrderNumberns.xml]: England-PrescriptionOrderUIDNumber

File: namingsystems/NamingSystem-England-PrescriptionOrderNumberuuidns.xml [hunk 9]
```diff
@@ -0,0 +1,25 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-PrescriptionOrderNumberuuidns"/>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-PrescriptionOrderNumberuuidns.xml]: England-PrescriptionOrderNumberUUID

**Review by KazeemHamzat — CHANGES_REQUESTED** [review]
Raise comments on the naming convention of elements NamingSystems.id and NamingSystems.name

File: namingsystems/NamingSystem-England-NhsMhsPartyKey.xml [hunk 10]
```diff
@@ -1,6 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-NhsMhsPartyKey.xml]: File name is yet to change to capital case.

Expect: namingsystems/NamingSystem-England-NHSMHSPartyKey.xml

File: namingsystems/NamingSystem-England-NhsSpineASID.xml [hunk 11]
```diff
@@ -1,6 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (KazeemHamzat) [inline namingsystems/NamingSystem-England-NhsSpineASID.xml]: File name: namingsystems/NamingSystem-England-NHSSpineASID.xml

**Review by KazeemHamzat — CHANGES_REQUESTED** [review]
Raise issues about lower case in file names

**Review by KazeemHamzat — DISMISSED** [review]
2 Observations:
1. Only 2 of the NamingSystems have thier OID Number included at this time. In the future version, can we find out the rest and include them?
2. Can you go on GitHub and click Resolve Conversation button for all the issues as they have now been resolved?

**Comment by SheenaSharma** [comment]
Changes are done.

File: namingsystems/NamingSystem-England-ERSHealthcareService.xml [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+            <rank value="1" />
+        </telecom>
+    </contact>
+    <responsible value="NHS England"/>
+    <description value="identifier of a healthcare service in eRS"/>
```
Comment (digitalhealthmartin) [inline namingsystems/NamingSystem-England-ERSHealthcareService.xml]: identifier -> Identifier

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

File: namingsystems/NamingSystem-England-ERSHealthcareService.xml (hunk 12 above)
Comment (SheenaSharma) [reply namingsystems/NamingSystem-England-ERSHealthcareService.xml]: Done with the changes.

**Review by digitalhealthmartin — APPROVED** [review]

**Review by KazeemHamzat — APPROVED** [review]

**Pull request merged**